onnxscript/tests/function_libs/torch_lib/ops_test_common.py

```diff
@@ -355,6 +355,41 @@ def _format_model_and_input_information(onnx_model, inputs):
     )
 
 
+TORCH_DTYPE_TO_ONNX_STRING = {
+    torch.bool: "tensor(bool)",
```

Comment:
nit: Commented elsewhere before but don't recall precisely. Does it has to be plain string for onnx types?

Reply:
Yes, it needs to be a plain string in onnx's type constraints

Reply:
I was wondering if there is anything like `str(onnx.BoolTensorType)`/`onnx.BoolTensorType.to_string()`

Reply:
I see. There probably is but we still need a mapping from torch types to onnx types. Any suggestions?